gtd-timer/gtdtimer.csproj

```diff
@@ -4,6 +4,11 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
probably got here by mistake

Reply:
I dig into this, and apparently this is a known issue: https://developercommunity.visualstudio.com/content/problem/319886/default-xml-documentation-file-path-is-absolute.html

but it can be fixed by manually setting 'XML documentation file' path in Project Properties -> Build:
![image](https://user-images.githubusercontent.com/45561295/49647600-bad72600-fa2b-11e8-89ce-548e350db2ff.png)

Reply:
I had this option already filled with .xml path. It seems it still produced that result.
![image](https://user-images.githubusercontent.com/12135899/49648992-7732eb00-fa30-11e8-9a8e-30ca1f38a529.png)

Reply:
edit that path manually - remove everything but file name from there. Result would be the same, but we'll get rid of absolute path in csproj